docs/user-guide/configure-load-balancing.md

```diff
@@ -9,25 +9,26 @@ Styx supports three load balancing strategies:
 Styx also provides a mechanism to bypass the load balancer and force
 the origin at source.
 
+##Load balancing strategies
```

Comment:
This heading is not correctly rendered.

Reply:
Oh thanks! It's rendering  fine inside IntelliJ, let me see if I can fix that...

Reply:
Hi. This is still not rendering correctly. Please address this.